x-pack/plugins/security_solution/public/common/components/charts/areachart.tsx

```diff
@@ -138,21 +145,47 @@ AreaChartBase.displayName = 'AreaChartBase';
 interface AreaChartComponentProps {
   areaChart: ChartSeriesData[] | null | undefined;
   configs?: ChartSeriesConfigs | undefined;
+  visualizationActionsOptions?: VisualizationActionsProps;
 }
 
-export const AreaChartComponent: React.FC<AreaChartComponentProps> = ({ areaChart, configs }) => {
+export const AreaChartComponent: React.FC<AreaChartComponentProps> = ({
+  areaChart,
+  configs,
+  visualizationActionsOptions,
+}) => {
   const { ref: measureRef, width, height } = useThrottledResizeObserver();
   const customHeight = get('customHeight', configs);
   const customWidth = get('customWidth', configs);
   const chartHeight = getChartHeight(customHeight, height);
   const chartWidth = getChartWidth(customWidth, width);
 
-  return checkIfAnyValidSeriesExist(areaChart) ? (
-    <WrappedByAutoSizer ref={measureRef} height={chartHeight}>
-      <AreaChartBase data={areaChart} height={chartHeight} width={chartWidth} configs={configs} />
-    </WrappedByAutoSizer>
-  ) : (
-    <ChartPlaceHolder height={chartHeight} width={chartWidth} data={areaChart} />
+  const isVlidSeriesExist = useMemo(() => checkIfAnyValidSeriesExist(areaChart), [areaChart]);
```

Comment:
typo here (`isValidSeriesExist`), I use the VSCode  extension: `streetsidesoftware.code-spell-checker`, it is very useful for me :)